src/components/15Years/15YearsButton.vue

```diff
@@ -0,0 +1,105 @@
+<script>
+export default {
+	render(createElement) {
+		const options = {
+			class: ['fifteen-yr-button', this.variant ? `fifteen-yr-button--${this.variant}` : ''],
```

Comment:
This component is a duplicate of KvButton but with a 'variant' prop and scoped styles. And a click handler override.